webextensions/api/webRequest.json

```diff
@@ -17,7 +17,9 @@
             "firefox_android": {
               "version_added": "48"
             },
-            "opera": "mirror",
+            "opera": {
```

Comment:
@queengooborg what's the reference for this update?

Reply:
At the time, I hadn't realize that the consistency linter was slightly broken when dealing with ranges, so I made some changes to Opera's data when it should've been Chrome that got updated.  If Opera is set to not mirror any longer (and I don't explicitly mention it), it's _probably_ wrong...!